### PR TITLE
Refactor: Applied styling changes for the Material TextBox

### PR DESCRIPTION
**Pull request by Massimo37**
GitHub Issue: #6 

## PR Type

What kind of change does this PR introduce?
Feature

## Description

Applied styling changes for the Material TextBox

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [X] Tested UWP
- [X] Tested iOS
- [X] Tested Android
- [X] Tested WASM
- [X] Contains **No** breaking changes

### CODE REVIEW

File: src/library/Uno.Material/Styles/Application/Colors.xaml [hunk 1]
```diff
@@ -132,6 +132,13 @@
 	<SolidColorBrush x:Key="OnSurfaceSelectedBrush"
 					 Color="{StaticResource OnSurfaceColor}"
 					 Opacity="0.08"/>
+	
+	<!-- Translucent Colors -->
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Application/Colors.xaml:136]: Do we really need those brushes? Let's at least use names from the color palette, not color01 :D

Comment (Massimo37) [reply src/library/Uno.Material/Styles/Application/Colors.xaml:136]: @MatFillion  Problem is that color exists in the TextBox style but not the brushes. What do we do ?

Comment (MatFillion) [reply src/library/Uno.Material/Styles/Application/Colors.xaml:136]: Let's keep it in TextBox.xaml then, use the name from the Variables in zeplin.

File: src/library/Uno.Material/Styles/Controls/TextBox.xaml [hunk 2]
```diff
@@ -54,14 +60,14 @@
 												Value="{StaticResource PrimaryColor01Brush}" />
 										<Setter Target="DeleteButton.Foreground"
 												Value="{StaticResource PrimaryColor01Brush}" />
+										<Setter Target="PlaceholderElement.Foreground"
+												Value="{StaticResource PrimaryColor01Brush}" />
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Controls/TextBox.xaml:64]: We shouldn't use PrimaryColor01Brush, use the new ones. In this case PrimaryBrush.

File: src/library/Uno.Material/Styles/Controls/TextBox.xaml [hunk 3]
```diff
@@ -42,7 +48,7 @@
 									<VisualState.Setters>
 										<Setter Target="ContentElement.Opacity"
 												Value="{StaticResource ControlPressedOpacity}" />
-										<Setter Target="PlaceholderTextElement.Opacity"
+										<Setter Target="PlaceholderElement.Opacity"
 												Value="{StaticResource ControlPressedOpacity}" />
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Controls/TextBox.xaml:52]: Is ControlPressedOpacity a system value?

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/App.xaml [hunk 4]
```diff
@@ -1,7 +1,8 @@
 ﻿<Application x:Class="Uno.Material.Samples.App"
 			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-			 xmlns:material="using:Uno.Material">
+			 xmlns:material="using:Uno.Material"
+			 RequestedTheme="Light">
```
Comment (MatFillion) [inline src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/App.xaml:5]: would that force light them? Not sure we want that!

File: src/library/Uno.Material/Styles/Application/Colors.xaml (hunk 1 above)
Comment (Massimo37) [reply src/library/Uno.Material/Styles/Application/Colors.xaml]: Done. Will add remaining TextBox colors and brushes in another PR.

File: src/library/Uno.Material/Styles/Controls/TextBox.xaml (hunk 3 above)
Comment (Massimo37) [reply src/library/Uno.Material/Styles/Controls/TextBox.xaml:56]: It's an old value that should be removed altogether from all controls.

File: src/library/Uno.Material/Styles/Controls/TextBox.xaml (hunk 2 above)
Comment (Massimo37) [reply src/library/Uno.Material/Styles/Controls/TextBox.xaml]: Fixed.

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/App.xaml (hunk 4 above)
Comment (Massimo37) [reply src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/App.xaml]: Debug code, forgot to remove :)

**Comment by agneszitte-nventive** [comment]
@Massimo37 missing conventional commit ;)

**Pull request merged**